src/sentry/hybridcloud/tasks/deliver_webhooks.py

```diff
@@ -71,7 +72,7 @@ class DeliveryFailed(Exception):
     queue="webhook.control",
     silo_mode=SiloMode.CONTROL,
 )
-def schedule_webhook_delivery(**kwargs) -> None:
```

Comment:
I think `**kwargs: Never` would make the type checker help more here (since we don't expect this to take args)